### PR TITLE
Add Navi32 Logic Yamls TN/NN BBS/HHS

### PR DESCRIPTION
**Pull request by wenchuanchen**
https://ontrack-internal.amd.com/browse/SWDEV-463142

### CODE REVIEW

**Review by jichangjichang — COMMENTED** [review]
except FP64 gemm or some special reason, we should not put pure gemm into Gridbased

**Review by cmingch — DISMISSED** [review]

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_BBS_BH_UserArgs.yaml [hunk 1]
```diff
@@ -0,0 +1,1691 @@
...
+  - - [128, 16, 1, 16, 128, 128, 128, 16]
+    - [2, 2.05378]
+  - - [128, 128, 1, 128, 128, 128, 128, 128]
+    - [5, 136.002]
+  - - [128, 2048, 1, 2048, 128, 128, 128, 2048]
+    - [3, 11173.2]
+- null
+- null
+- DeviceEfficiency
+- GridBased
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/navi32/Equality/navi32_Cijk_Ailk_Bljk_BBS_BH_UserArgs.yaml]: Equality

**Review by jichangjichang — DISMISSED** [review]
lgtm

**Review by cmingch — DISMISSED** [review]

**Review by cmingch — APPROVED** [review]

**Comment by wenchuanchen** [comment]
For hipblaslt-test:
[hipblaslt-test.log](https://github.com/user-attachments/files/16721034/hipblaslt-test.log)

[----------] Global test environment tear-down
[==========] 12325 tests from 13 test suites ran. (1361087 ms total)
[  PASSED  ] 12325 tests.
hipBLASLt version: 1000

**Comment by wenchuanchen** [comment]
For tox:
================================== 9 passed, 84 skipped, 36 warnings in 1719.63s (0:28:39) ===================================
py3: exit 0 (1719.84 seconds) /dockerx/hipBLASLt/tensilelite> py.test -v --basetemp=/tmp/.tensile-tox/py3/tmp --junit-xml=/dockerx/hipBLASLt/tensilelite/python_tests.xml --junit-prefix=py3 --color=yes -n 4 --prebuilt-client=/tmp/.tensile-tox/py3/client/0_Build/client/tensile_client Tensile/Tests -m common pid=491872
  py3: OK (1858.42=setup[11.65]+cmd[126.92,1719.84] seconds)
  congratulations :) (1858.46 seconds)

**Review by hcman2 — APPROVED** [review]

**Review by jichangjichang — APPROVED** [review]
lgtm

**Review by Serge45 — APPROVED** [review]

**Pull request merged**